python/paddle/tensor/math.py

```diff
@@ -7137,3 +7138,52 @@ def combinations(x, r=2, with_replacement=False, name=None):
         grids[i] = grids[i].masked_select(mask)
 
     return paddle.stack(grids, 1)
+
+
+def signbit(x, name=None):
+    r"""
+    Tests if each element of input has its sign bit set or not.
+
+    Args:
+        x (Tensor): The input Tensor. Must be one of the following types: float16, float32, float64, bfloat16, uint8, int8, int16, int32, int64.
+        name (str, optional): Name for the operation (optional, default is None).For more information, please refer to :ref:`api_guide_Name`.
+
+    Returns:
+        out (Tensor): The output Tensor. The sign bit of the corresponding element of the input tensor, True means negative, False means positive.
+            >>> # example1
+            >>> x = paddle.to_tensor([1.1, -2.1, 0., 2.5], dtype='float32')
+            >>> res = paddle.signbit(x, y)
+            >>> print(res)
+            Tensor(shape=[4], dtype=bool, place=Place(cpu), stop_gradient=True,
+            [False , True, False, False])
+
+            >>> # example2
+            >>> x = paddle.to_tensor([-5, -2, 3], dtype='int32')
+            >>> res = paddle.signbit(x, y)
+            >>> print(res)
+            Tensor(shape=[3], dtype=bool, place=Place(cpu), stop_gradient=True,
+            [True. , True , False])
```

Comment:
should add example code to decribe the result of `-0.0` or `-0`

Reply:
done